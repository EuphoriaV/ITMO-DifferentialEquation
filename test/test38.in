2.7
(((x * (x + 7.23)) - (x * 5.23)) + (((x - (10.2 + ((7.57 * 5.21) - ((x * x) * 2.94)))) + x) - ((x * (4.98 - 2.65)) * (x + x))))
3.85
1.64