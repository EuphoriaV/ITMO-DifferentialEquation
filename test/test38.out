-4.673E-4